### PR TITLE
Increase MFTF coverage.

### PR DESCRIPTION
**Pull request by nmalevanec**
### Description (*)

Automated tests:
1. https://app.hiptest.com/projects/69435/test-plan/folders/457016/scenarios/1820455
2. https://app.hiptest.com/projects/69435/test-plan/folders/457016/scenarios/1549558
3. https://app.hiptest.com/projects/69435/test-plan/folders/481999/scenarios/1652313
4. https://app.hiptest.com/projects/69435/test-plan/folders/481999/scenarios/1655285
5. https://app.hiptest.com/projects/69435/test-plan/folders/735282/scenarios/2076883
6. https://app.hiptest.com/projects/69435/test-plan/folders/735257/scenarios/2126218
7. https://app.hiptest.com/projects/69435/test-plan/folders/735167/scenarios/2427051
8. https://app.hiptest.com/projects/69435/test-plan/folders/735234/scenarios/1439899
9. https://app.hiptest.com/projects/69435/test-plan/folders/735182/scenarios/2551702
10. https://app.hiptest.com/projects/69435/test-plan/folders/735182/scenarios/1477876

Automated Reorder tests:
1. https://app.hiptest.com/projects/69435/test-plan/folders/457017/scenarios/1550639
2. https://app.hiptest.com/projects/69435/test-plan/folders/735202/scenarios/1482505
3. https://app.hiptest.com/projects/69435/test-plan/folders/735202/scenarios/1676763
4. https://app.hiptest.com/projects/69435/test-plan/folders/735202/scenarios/2591613
5. https://app.hiptest.com/projects/69435/test-plan/folders/735202/scenarios/2591627
6. https://app.hiptest.com/projects/69435/test-plan/folders/735202/scenarios/2591657
7. https://app.hiptest.com/projects/69435/test-plan/folders/735202/scenarios/1482457

Additionally automated tests:
1. https://app.hiptest.com/projects/69435/test-plan/folders/481999/scenarios/3334786 
2. https://app.hiptest.com/projects/69435/test-plan/folders/457016/scenarios/1699941 
3. https://app.hiptest.com/projects/69435/test-plan/folders/457017/scenarios/1620000 
4. https://app.hiptest.com/projects/69435/test-plan/folders/457017/scenarios/1699962 
5. https://app.hiptest.com/projects/69435/test-plan/folders/457017/scenarios/2197183 
6. https://app.hiptest.com/projects/69435/test-plan/folders/735182/scenarios/2553867 
7. https://app.hiptest.com/projects/69435/test-plan/folders/735236/scenarios/1694896 
8. https://app.hiptest.com/projects/69435/test-plan/folders/735207/scenarios/1677021 
9. https://app.hiptest.com/projects/69435/test-plan/folders/735287/scenarios/1583150 
10. https://app.hiptest.com/projects/69435/test-plan/folders/735178/scenarios/2562634
11. https://app.hiptest.com/projects/69435/test-plan/folders/735226/scenarios/1698409
12. https://app.hiptest.com/projects/69435/test-plan/folders/735224/scenarios/2620399
13. https://app.hiptest.com/projects/69435/test-plan/folders/735141/scenarios/1695035
14. https://app.hiptest.com/projects/69435/test-plan/folders/735141/scenarios/2129631
15. https://app.hiptest.com/projects/69435/test-plan/folders/735141/scenarios/2487925
16. https://app.hiptest.com/projects/69435/test-plan/folders/735203/scenarios/2601672
17. https://app.hiptest.com/projects/69435/test-plan/folders/419536/scenarios/2941524
18. https://app.hiptest.com/projects/69435/test-plan/folders/419537/scenarios/2186548
19. https://app.hiptest.com/projects/69435/test-plan/folders/735236/scenarios/1443923
20. https://app.hiptest.com/projects/69435/test-plan/folders/735182/scenarios/2553502

### Fixed Issues (if relevant)
1. magento/inventory#2523: Increase MFTF coverage.
2. magento/inventory#2516: Reorder With Simple Product automation

### Manual testing scenarios (*)

1. none.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by ishakhsuvarov** [comment]
@magento run Functional Tests build

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateBundleProductAdditionalStockAdditionalWebsiteTest.xml [hunk 1]
```diff
@@ -121,6 +121,6 @@
         <amOnPage url="{{StorefrontHomePage.url}}/{{customStore.code}}/$$category.name$$.html" stepKey="navigateToCategoryPage"/>
         <waitForPageLoad stepKey="waitForCategoryPageLoad"/>
         <see userInput="{{BundleProduct.name}}" stepKey="assertProductPresent"/>
-        <see userInput="{{BundleProduct.price}}" stepKey="assertProductPricePresent"/>
+        <see userInput="$615.00" stepKey="assertProductPricePresent"/>
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateBundleProductAdditionalStockAdditionalWebsiteTest.xml]: Is this a typo or it makes sense to hardcode?

**Comment by ishakhsuvarov** [comment]
@magento run Functional Tests build

**Comment by ishakhsuvarov** [comment]
@nmalevanec Please check the failing test. I've encountered multiple occasions when it would fail without a clear reason. It might need to be tuned.

```
https://app.hiptest.com/projects/69435/tests-plan/folders/419537/scenarios/1437519: Logged in Customer ordered Grouped product with child products assigned to default Stock from Homepage
```
(or [here](https://testing-service.magento-community.engineering/reports/magento/inventory/pull/2524/b690ecb3e0363f1b582edcf5982a2adc/Functional/allure-report-ce/index.html#categories/aa265569ca89ddb0e7fbe294cdc8ce73/f1c8155e581267b4/))

I'll restart builds to see if it helps

**Comment by ishakhsuvarov** [comment]
@magento run Functional Tests build

**Review by ishakhsuvarov — CHANGES_REQUESTED** [review]
Please refactor assertions which rely on the price directly inside of the test. Automated tests are not supposed to contain those values and are not supposed to do the calculation in the runtime as this practice is error-prone.

Please use the following as examples:
https://github.com/magento/magento-functional-tests-migration/blob/2.3-develop/app/code/Magento/Catalog/Test/Mftf/Data/ProductData.xml#L17

https://github.com/magento/magento-functional-tests-migration/blob/2.3-develop/app/code/Magento/Catalog/Test/Mftf/Test/AdminCreateProductCustomAttributeSet.xml#L85

File: app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateBundleProductAdditionalStockAdditionalWebsiteTest.xml (hunk 1 above)
Comment (nmalevanec) [reply app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminCreateBundleProductAdditionalStockAdditionalWebsiteTest.xml]: Price $615 is consists of bundle product price itself and option price. There is no math operations in MFTF as far as I know

**Comment by ishakhsuvarov** [comment]
@magento run Functional Tests build

**Review by ishakhsuvarov — APPROVED** [review]

**Pull request merged**